File: Pod/Classes/FNKBarSectionData.m
//
//  FNKBarSectionData.m
//  Pods
//
//  Created by Phillip Connaughton on 1/12/15.
//
//

#import "FNKBarSectionData.h"

@implementation FNKBarSectionData

+(FNKBarSectionData*)barSectionData:(id)data bucket:(int)bucket
{
    FNKBarSectionData* bsd = [[FNKBarSectionData alloc] init];
    [bsd setData:data];
    [bsd setBucket:bucket];
    return bsd;
}

@end

File: Pod/Classes/FNKBarSectionData.swift
//
//  FNKBarSectionData.swift
//  Pods
//
//  Created by Phillip Connaughton on 11/21/15.
//
//

import Foundation

class FNKBarSectionData
{
    let data: AnyObject
    let bucket: Int
    
    init(data: AnyObject, bucket: Int)
    {
        self.data = data
        self.bucket = bucket
    }
}